spark/dl/src/main/scala/com/intel/analytics/bigdl/example/treeLSTMSentiment/README.md

```diff
@@ -0,0 +1,68 @@
+## Summary
+This example shows how to use BigDL train a model on [Standford Treebank
+dataset](https://nlp.stanford.edu/sentiment/index.html) dataset using binary TreeLSTM and [Glove](https://nlp.stanford.edu/projects/glove/)
+word embedding vectors.   Tree-LSTM is a kind of recursive neural networks, which describes in the paper 
+[Improved Semantic Representations From Tree-Structured Long Short-Term Memory Networks](https://arxiv.org/abs/1503.00075)
+ by Kai Sheng Tai, Richard Socher, and Christopher Manning.
+
+The dataset is a corpus of ~10K one-sentence movie reviews from Rotten Tomatoes. Each sentence has been parsed into
+[constituency-based parse trees](https://en.wikipedia.org/wiki/Parse_tree#Constituency-based_parse_trees), which is
+kind of binary trees with a word at each leaf. After pre-processing, every node has been tagged a label whose range from -2 to 2, representing 
+the sentiment of the word or phrase. The value from -2 to 2 corresponds to highly negative, moderately negative, neutral, moderately positive and
+highly positive respectively. The root of the tree represents the sentiment of the entire sentence.
+
+## Steps to run this example:
+First run the following script
+
+```{r, engine='sh'}
+python fetch_and_preprocess.py
+```
+
+The treebank dataset and the Glove word embedding vectors will be downloaded to
+`/tmp/.bigdl/dataset/` directory, after that the treebank will be split into three folders
+corresponding to train, dev, and test in an appropriate format.
+
+Next just run the following command to run the code:
+
+* Spark local:
+
+```{r, engine='sh'}
+     spark-submit --master "local[physical_core_number]" --driver-memory 20g      \
```

Comment:
there will be some error.Exception in thread "main" java.io.FileNotFoundException: /tmp/.bigdl/dataset/sst/vocab-cased.txt (No such file or directory)